poleno/utils/tests/test_templatetags.py

```diff
@@ -238,9 +238,9 @@ def test_active_filter(self):
     def test_change_lang_tag(self):
         u"""
         Tests ``change_lang`` template tag by requesting a view using a template with this tag.
-        Checking that it generates corrent links to the same view in different languages.
+        Checking that it generates correct links to the same view in different languages.
         """
-        lang = ((u'de', u'Deutsch'), ('en', u'English'), ('fr', u'Francais'))
+        lang = ((u'de', u'Deutsch'), (u'en', u'English'), (u'fr', u'Francais'))
```

Comment:
@martinmacko47 Je potrebne pri zmenej funkcionalite tagu `change_lang` aj doplnit zmenenu funkcionalitu aj do testov?

Reply:
@viliambalaz Aj hej. Len mi nenapada ako. Lebo v testoch v adresari `/poleno` nemozeme mat 404 template z aplikacie. Takze ked pouzijes `self.client.get(u'/language/')`, tak to `change_lang` vobec nezavola a iba vrati redirect na `/en/language/`.

Ak by sme spravili nejaky nelokalizovany endpoint, tak ten by sa sice neredirectol, ale na nom by `change_lang` fungovalo aj bez fixnutia.

Jedine, co mi napada je spravit si vlastny 404 template, kde sa pouzuje `change_lang`. A teda bez fixu `self.client.get(u'/language/')` padne na errore v tem 404 template. Ak sa ti chce, mozes skusit.

Reply:
:ok: https://github.com/martinmacko47/chcemvediet/issues/280#issuecomment-706953197, upravim to v samostatnom PR.